docker/run-backend.sh

```diff
@@ -1,11 +1,27 @@
 #!/bin/bash -ex
 
+brotliFolder=".brotli"
+
 ./telemetry.sh
 source env.sh
 
 # start backend
 pushd ../backend
 go install github.com/cosmtrek/air@latest
 go install github.com/go-delve/delve/cmd/dlv@latest
+
+
+if [ ! /usr/local/bin/brotli ]; then
```

Comment:
weird, this check didn't enter but i don't have the file
<img width="552" alt="Screenshot 2023-11-17 at 9 52 09 AM" src="https://github.com/highlight/highlight/assets/1351531/b7eaab62-1dcd-4099-ac98-6b17e1922a72">
<img width="578" alt="Screenshot 2023-11-17 at 9 52 26 AM" src="https://github.com/highlight/highlight/assets/1351531/1af41877-3e63-4e9a-ba5f-49c3d6d55d3a">